tests/lcs/core/execution/uql.livecodescript

```diff
@@ -0,0 +1,36 @@
+﻿script "CoreExecutionUQL"
+/*
+Copyright (C) 2018 LiveCode Ltd.
+
+This file is part of LiveCode.
+
+LiveCode is free software; you can redistribute it and/or modify it under
+the terms of the GNU General Public License v3 as published by the Free
+Software Foundation.
+
+LiveCode is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or
+FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+for more details.
+
+You should have received a copy of the GNU General Public License
+along with LiveCode.  If not see <http://www.gnu.org/licenses/>.  */
+
+on TestUQLPut
+	put "foo" into tTest1
+	TestAssert "unquoted literal put into", tTest1 is "foo"
+	put "foo" after tTest2
+	TestAssert "unquoted literal put after", tTest2 is "foo"
+	put "foo" before tTest3
+	TestAssert "unquoted literal put before", tTest3 is "foo"
```

Comment:
@runrevmark when you get a chance could you review these tests as I don't recall when the change was made (if it was deliberate) for UQLs to be cleared of their value in `put before|after` situations. Previously if doing something like appending to a variable you were required to `put empty into var` before you started appending to it otherwise you got the var name included in the value.

As in:
```
put "foo" after bar
-- bar is now "barfoo"
```
But now:
```
put "foo" after bar
-- bar is now "foo"
```

Reply:
It would have been a long time ago - certainly 7.0 at the earliest...

Indeed https://quality.livecode.com/show_bug.cgi?id=14536 was fixed in a 7 release and it basically put the clear_uql() call into the matchtext/binaryDecode which seems to have disappeared since (where you have put it is a more sensible place).

My guess is that when we containerized variables the refactoring missed the need for clearuql in those two places.

Reply:
This goes back to before 6.7.11 as well - MCVarref's used to do a clearuql before they were accessed - so *not* clearing them was probably fixed a long time ago.